packages/stops-overlay/src/index.tsx

```diff
@@ -143,9 +148,9 @@ const StopsOverlay = (props: Props): JSX.Element => {
           id="stops"
           minzoom={minZoom || 10}
           paint={{
-            "circle-color": "#fff",
+            "circle-color": color || "#fff",
             "circle-opacity": 0.9,
-            "circle-stroke-color": "#333",
+            "circle-stroke-color": color ? "#fff" : "#333",
```

Comment:
Can we put a todo here to use tinycolor or similar to generate an appropriate stroke color